backend/tests/integration/search/test_api.py

```diff
@@ -901,7 +901,7 @@ def test_journal_title_normalization_without_match(inspire_app):
 
 
 def test_public_api_generates_correct_links_in_literature_search_with_fields(
-    inspire_app
+    inspire_app,
```

Comment:
are these changes because of your editor settings?

Reply:
It's pre-commit, but I've just fixed it ;)